UPGRADE-2.8.md

```diff
@@ -511,5 +511,5 @@ Yaml
 
  * Deprecated usage of a colon in an unquoted mapping value
  * Deprecated usage of `@`, `` ` ``, `|`, and `>` at the beginning of an unquoted string
- * Deprecated non-escaped \ in double-quoted strings when parsing Yaml
-   ("Foo\Var" is not valid whereas "Foo\\Var" is)
+ * Deprecated non-escaped `\` in double-quoted strings when parsing Yaml
+   (`Foo\Var` is not valid whereas `Foo\\Var` is)
```

Comment:
Can you update the note. How about this (FYI: I'm missing a closing tick after the code-blocks, so it'll render here)

```md
When surrounding strings with double-quotes, you must now escape `\` characters. Not escaping
those characters (when surrounded by double-quotes) is deprecated.

Before:

```yml
class: "Foo\Var"
``

After:

```yml
class: "Foo\\Var"
``
```